src/main/java/com/teamrocket/tms/services/team/TeamServiceImpl.java

```diff
@@ -25,10 +25,11 @@ public TeamServiceImpl(TeamRepository teamRepository, ModelMapper modelMapper, T
     }
 
     @Override
-    public TeamDTO createTeam(TeamDTO teamDTO) {
+    public TeamDTO createTeam(TeamDTO teamDTO, String creatorName) {
         teamServiceValidation.validateTeamAlreadyExists(teamDTO);
 
         Team team = modelMapper.map(teamDTO, Team.class);
+        team.setTeamLeader(creatorName);
```

Comment:
As of the making of this post there is conflicting info on our diagram in 2 places regarding team creation

Please consider:
- if only Project Manager can create a Team and teamLeader is set to creator, it makes it complicated to set another Team Leader
- created Team should just have a name on creation

Reply:
Sure! But It was mentioned in the Task description on Trello that the team leader should be auto set to the creator. Will refactor it.